src/core/generate.ml

```diff
@@ -161,8 +161,12 @@ let axiom (a : Translated.axiom) =
   in
   [ [%stri let () = [%e body]] ]
 
-let structure driver : structure =
+let structure runtime driver : structure =
   (pmod_ident (lident (Drv.module_name driver)) |> include_infos |> pstr_include)
+  :: pstr_module
+       (module_binding
+          ~name:{ txt = Some "Runtime"; loc }
+          ~expr:(pmod_ident (lident runtime)))
```

Comment:
That's `module Runtime = <runtime>` right?
Perhaps `Runtime` is a bit too invasive as It prevents the user to call a module like this. Why did you move away from `Ortac_runtime`?

Reply:
You're right. I'll fix this.